src/expandableCalendar/agendaList.tsx

```diff
@@ -187,6 +188,7 @@ const AgendaList = (props: AgendaListProps) => {
         }
       }
     }
+    onViewableItemsChanged?.(info);
```

Comment:
I'd consider adding the `onViewableItemsChanged` to the deps array

Reply:
Done